tools/generators/dependency-mismatch/index.spec.ts

```diff
@@ -0,0 +1,175 @@
+import { createTreeWithEmptyWorkspace } from '@nrwl/devkit/testing';
+import { Tree, addProjectConfiguration, serializeJson, readWorkspaceConfiguration, readJson } from '@nrwl/devkit';
+
+import generator from './index';
+import { PackageJson } from '../../types';
+
+describe('release-dependency-mismatch generator', () => {
```

Comment:
nit: should probably have this match the name of the generator
```suggestion
describe('dependency-mismatch generator', () => {
```